_posts/zh/newsletters/2019-04-30-newsletter.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Bitcoin Optech Newsletter #44'
+permalink: /zh/newsletters/2019/04/30/
+name: 2019-04-30-newsletter-zh
+slug: 2019-04-30-newsletter-zh
+type: newsletter
+layout: newsletter
+lang: zh
+---
+本周的 Newsletter 再次迎来了一个新闻较少的周，但仍包含了关于 bech32 发送支持、Bitcoin Stack Exchange 精选问答以及流行的 Bitcoin 基础设施项目中的值得注意的变化的常规部分。
+
+{% include references.md %}
+
+## 行动项
+
+*撰写时无事项。* 注意：如果 Bitcoin Core 发布团队认为上周分发的第四个[候选版本][0.18.0]中没有发现阻塞问题，他们打算在本周发布此 Newsletter 时标记最终版本 0.18.0。如果发生这种情况，我们将在下周的 Newsletter 中提供详细的版本发布报道。但如果你计划升级，请不要等待我们——所有你需要了解的新版本信息都在其发行说明中解释，这些说明将在不同平台（如 [BitcoinCore.org][]）的各种发布公告中发布或链接。
+
+## 新闻
+
+*本周无新闻。希望大家享受一个美好的春天、秋天或旱季。*
+
+## Bech32 发送支持
+
+*第 7 周，共 24 周。在 2019 年 8 月 24 日 segwit 软分叉锁定的二周年纪念日前，Optech Newsletter 将包含此每周部分，为开发人员和组织提供有关实现 bech32 发送支持（支付本机 segwit 地址的能力）的信息。这[不需要实现 segwit][bech32 series] 本身，但确实允许你支付的人访问 segwit 的所有多个好处。*
+
+{% comment %}<!-- weekly reminder for harding: check Bech32 Adoption
+wiki page for changes -->{% endcomment %}
+
+{% include specials/bech32/07-altbech32.md %}
```

Comment:
Should be zh version

Reply:
I encountered some issues with the merge operations after using `git rebase`. I have identified and corrected my mistakes. However, I am still not very proficient with the merge process after git rebase, so I will pay more attention to these details in the future. Thank you very much for your review and assistance!